lnwallet/channel.go

```diff
@@ -9160,13 +9195,20 @@ func CreateCooperativeCloseTx(fundingTxIn wire.TxIn,
 		fundingTxIn.Sequence = mempool.MaxRBFSequence
 	}
 
+	// Otherwise, a custom sequence might be specified.
+	opts.customSequence.WhenSome(func(sequence uint32) {
+		fundingTxIn.Sequence = sequence
+	})
+
 	// Construct the transaction to perform a cooperative closure of the
 	// channel. In the event that one side doesn't have any settled funds
 	// within the channel then a refund output for that particular side can
 	// be omitted.
 	closeTx := wire.NewMsgTx(2)
 	closeTx.AddTxIn(&fundingTxIn)
 
+	// TODO(roasbeef): needs support for dropping inputs
```

Comment:
Is this TODO supposed to be addressed in this series or is it for posterity?

Reply:
Dunno....removed. 

I think maybe it was a TODO I added during dev to remember that we need to look at the 3 diff sig types to know which outputs we should add to the transaction.